Sources/SnapshotTesting/Snapshotting.swift

```diff
@@ -46,8 +46,44 @@ public struct Snapshotting<Value, Format> {
     }
   }
 
+  /// Transforms a strategy on `Value`s into a strategy on `NewValue`s through a function
+  /// `(NewValue) -> Value`.
+  ///
+  /// This is the most important operation for transforming existing strategies into new strategies. It
```

Comment:
For docs I've been letting Xcode wrap paragraphs on single lines. Just seems annoying to have to manage the line breaks of prose as it gets updated. Down to pick either lane, though. We should just be consistent.

Reply:
I just like it for the PR readability. also if we were to ever lint our oss (which might be necessary with contributors) I image we would want a line length rule.